helpers/dbr.js

```diff
@@ -0,0 +1,4 @@
+import DBR from 'dynamsoft-javascript-barcode';
+DBR.BarcodeReader.engineResourcePath =
+  'https://cdn.jsdelivr.net/npm/dynamsoft-javascript-barcode@8.8.7/dist/';
```

Comment:
`package.json` has `"dynamsoft-javascript-barcode": "^8.8.7"`. Seems fragile that when 8.8.8 is released, it will be installed, but this path won't get updated.  Should the package file lock to a specific version?  Or can this path be dynamically set?

Reply:
I suppose its best to lock the package by not including the `^` in this situation.